action.yml

```diff
@@ -6,8 +6,16 @@ branding:
   icon: 'settings'
   color: 'gray-dark'
 
+inputs:
+  directories:
+    description: 'Comma-separated list of directories to look for coverage information (e.g. `src,examples`)'
+    required: false
+    default: 'src'
+
 runs:
   using: 'composite'
   steps:
     - run: julia --color=yes "$GITHUB_ACTION_PATH"/main.jl
       shell: bash
+      env:
+        - INPUT_DIRECTORIES
```

Comment:
This line seems to be wrong: https://github.com/trixi-framework/Trixi.jl/pull/630/checks?check_run_id=2745439242#step:1:34

But I am not an expert  enough on GHA scripting to tell you how to fix it :-/